agasc/agasc.py

```diff
@@ -182,7 +240,7 @@ def add_pmcorr_columns(stars, date):
 
 
 def get_agasc_cone(ra, dec, radius=1.5, date=None, agasc_file=None,
-                   pm_filter=True, fix_color1=True, use_mag_est=False):
+                   pm_filter=True, fix_color1=True, use_supplement=False):
```

Comment:
As we dig more into implementation... what would the disadvantages be at this point to flipping the default?

Reply:
Basically any existing code that uses AGASC mags will no longer give reproducible results. I'm a bit wary of that.

Reply:
Right, I completely agree it is a scary idea, but I'm not convinced there is code out there for which it matters.

Reply:
OK, let's not be so timid! I guess there is precedent with fixing star colors by default. I have also globally enabled this by default.